velox/functions/prestosql/GeometryFunctions.h

```diff
@@ -1951,7 +1952,7 @@ struct StLineFromTextFunction {
     GEOS_TRY(
         {
           geos::io::WKTReader reader;
-          geosGeometry = reader.read(wkt);
+          geosGeometry = reader.read(std::string(wkt));
```

Comment:
fixme: We don't want to do this. These strings are often large (MBs). The reader takes a const std::string &. Can we make an explicit cast to that? Or skip this one?

Reply:
@jagill ditto